lib/plausible_api/event/post.rb

```diff
@@ -114,10 +92,9 @@ def blank?(value)
         !present?(value)
       end
 
-      def maybe_string(value, default: nil)
-        return default if value.nil?
-        value = value.to_s.strip
-        value.empty? ? default : value
+      def presence(value)
```

Comment:
The only difference here is losing the strip, but it's probably ok.

Reply:
@jnunemaker you are totally right, but I thought that striping shouldn't be responsability of the gem. We should include it for any parameter to the stats api then...

Reply:
Ok!